docs/source/reference/northstar/serversiderui.rst

```diff
@@ -0,0 +1,187 @@
+
+**Example:**
+
+.. code-block:: javascript
+
+    void function CreateDummyPoll()
+     {
+        array<string> options = [ "Me", "You", "Both of us" ]
+        foreach(entity player in GetPlayerArray())
+            NSCreatePollOnPlayer(player, "Vote who's the biggest dummy!", options, 30)
+      }
+
+Getting Response
+-----
+
+**Definition:**
+
+.. cpp:function:: int NSGetPlayerResponse( entity player )
+
+Returns the index of the item from ``options`` the player voted for. If the player hadn't voted yet it returns a -1.
+
+**Example:**
+
+.. code-block:: javascript
+
+    void function CheckResponseToDummyPoll(entity player)
+    {
+        if(NSGetPlayerResponse(player) != -1)
+            print("Player has voted!")
+    }
+
+Large Message
+^^^^^
+
+Sends a large message to ``player`` which will appear in the top right corner.
+
+.. figure:: /_static/serversiderui/largemessage.gif
+  :align: center
+  :class: screenshot
+
+  Player POV
+  
+**Definition:**
+
+.. cpp:function:: void NSSendLargeMessageToPlayer( entity player, string title, string description, float duration, string image )
+
+**Example:**
+
+.. code-block:: javascript
+
+    void function SendDummyLargeMessage(entity player)
+    {
+        NSSendLargeMessageToPlayer(player,"I'm not a dummy >:(", "You are", 10, "ui/fd_tutorial_tip.rpak")
+    }
+
+Info Message
+^^^^^
+
+Sends a smaller message to ``player`` which will appear from the center right.
+
+.. figure:: /_static/serversiderui/info.gif
+  :align: center
+  :class: screenshot
+
+  Player POV
+
+**Definition:**
+
+.. cpp:function:: void NSSendInfoMessageToPlayer( entity player, string text )
+
+**Example:**
+
+.. code-block:: javascript
+  
+    void function SendDummyInfoMessage(entity player)
+    {
+        NSSendInfoMessageToPlayer(player, "If you're reading this your a dummy")
+    }
+
+PopUp
+^^^^^
+
+Send a small popup to ``player`` which will appear in the lower half of their screen under their cursor.
+
+.. figure:: /_static/serversiderui/popup.gif
+  :align: center
+  :class: screenshot
+
+  Player POV
+
+**Definition:**
+
+.. cpp:function:: void function NSSendPopUpMessageToPlayer( entity player, string text )
+
+**Example:**
+
+.. code-block:: javascript
+
+    void funcions SendDummyPopUp(entity player)
+    {
+        NSSendPopUpMessageToPlayer(player, "Dummy")
+    }
+
+Announcement
+^^^^^
+
+Sends a large announcement to ``player``.
+
+.. figure:: /_static/serversiderui/announcement.gif
+  :align: center
+  :class: screenshot
+
+  Player POV
+
+**Definition:**
+
+.. cpp:function:: void NSSendAnnouncementMessageToPlayer( entity player, string title, string description, vector color, int priority, int style )
+
+**Example:**
+
+.. code-block:: javascript
+
+      void function SendDummyAnnouncement(entity player)
+      {
+          NSSendAnnouncementMessageToPlayer(player, "Large dummy", "Small dummy", <1,1,0>, 1, ANNOUNCEMENT_STYLE_QUICK)
```

Comment:
Again I'd suggest using the same text as in the example GIF to make the mapping clearer.